LPS-91032 BuildLang

veroglez opened the pull request.

liferay-echo commented:
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-12.liferay.com/job/test-portal-source-format/5728/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/5728/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0848120527e55ff5257496f80e8881ee0ddbaff6">0848120527e55ff5257496f80e8881ee0ddbaff6</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/veroglez/liferay-portal/tree/LPS-91032">LPS-91032</a><br/>Branch GIT ID: <a href="https://github.com/veroglez/liferay-portal/commit/dcccb3d6996f08a16910d604eb538b0f7918330b">dcccb3d6996f08a16910d604eb538b0f7918330b</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-12.liferay.com/job/test-portal-source-format/5728/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/5728/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-12.liferay.com/job/test-portal-source-format/5728/">test-portal-source-format#5728</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/5728/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-echo/liferay-portal/pull/14752">liferay-echo#14752</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2476012808">[master] ci:test:sf - veroglez &gt; liferay-echo - PR#14752 - 2023-12-18[09:28:21]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-12.liferay.com/job/publish-testray-report/4465/">publish-testray-report#4465</a><br/></div></div>

sandrodw3 commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-5.liferay.com/job/test-portal-source-format/6068/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-5.liferay.com/userContent/jobs/test-portal-source-format/builds/6068/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/06eeef7207154be1ba6c215c260f89565d6e7eca">06eeef7207154be1ba6c215c260f89565d6e7eca</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/veroglez/liferay-portal/tree/LPS-91032">LPS-91032</a><br/>Branch GIT ID: <a href="https://github.com/veroglez/liferay-portal/commit/8ef558e780ae30d380377c4b812a8c7cc2e60cd1">8ef558e780ae30d380377c4b812a8c7cc2e60cd1</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-5.liferay.com/job/test-portal-source-format/6068/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-5.liferay.com/userContent/jobs/test-portal-source-format/builds/6068/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-5.liferay.com/job/test-portal-source-format/6068/">test-portal-source-format#6068</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-5.liferay.com/userContent/jobs/test-portal-source-format/builds/6068/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-echo/liferay-portal/pull/14752">liferay-echo#14752</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2476442199">[master] ci:test:sf - veroglez &gt; liferay-echo - PR#14752 - 2023-12-19[03:27:17]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-5.liferay.com/job/publish-testray-report/2680/">publish-testray-report#2680</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/8053/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8053/jenkins-report.html">here</a>.</b></p>

ethib137 commented:
Hey @veroglez , this shouldn't require any new language keys. We just need to copy the existing pattern we have for this. You can check the [Fix Notes](https://liferay.atlassian.net/browse/LPS-91032) I added in the ticket description to see how it should be done. Let me know if you have any questions.

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/8053/">test-portal-acceptance-pullrequest(master)#8053</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8053/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-echo/liferay-portal/pull/14752">liferay-echo#14752</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2476468718">[master] ci:test:relevant - veroglez &gt; liferay-echo - PR#14752 - 2023-12-19[03:43:45]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/15945/">publish-testray-report#15945</a><br/></div></div>

veroglez commented:
> Hey @veroglez , this shouldn't require any new language keys. We just need to copy the existing pattern we have for this. You can check the [Fix Notes](https://liferay.atlassian.net/browse/LPS-91032) I added in the ticket description to see how it should be done. Let me know if you have any questions.

Hi @ethib137! analyzing the localized input pattern, I found some details that could be improved and I added them to this selector:
- Add a tooltip with the action since the action is not visually identifiable, the same pattern of the icon button. For that reason I added a new language key. If you want, we can remove it from this PR and open an improvement proposal to add it to all language selectors.
- Regarding the Fix Notes, I think we can improve the phrase, [following this document](https://liferay.atlassian.net/wiki/spaces/PEDS/pages/2253291959/Triggering+actions+in+components+pattern) 
    - We can put the action first, and the additional information later: “Select language, current language x”
    - We can also remove the specific mention of “enter” since this control can be activated by space/enter, key combination required by the screen reader, or by mouse click/touch. 

What do yo think? If you want we can also create a proposal to improve the language selector of the localized input.

ethib137 commented:
Hey @veroglez , I definitely see value in the changes you are suggesting. I think it will be best if we follow your suggestion to create a new ticket for those improvements, where we can apply them consistently across all language selectors. So for this PR, let's keep the language keys the same. The other improvements should be fine. Just for the tooltip can we make it `Select a Language` instead of `Select Language`. Does that sound alright?

For the new Ticket we can then have @marcoscv-work review it as well.

veroglez commented:
Thanks for your feedback @ethib137! I have updated the code with the change of the language key according to your suggestion and created a task for the accessibility improvement of the language selector https://liferay.atlassian.net/browse/LPS-204398 (cc @marcoscv-work)

veroglez commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/8073/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8073/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-12.liferay.com/job/test-portal-source-format/5742/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/5742/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b808cb100001223008944254d5da1364de64c622">b808cb100001223008944254d5da1364de64c622</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/veroglez/liferay-portal/tree/LPS-91032">LPS-91032</a><br/>Branch GIT ID: <a href="https://github.com/veroglez/liferay-portal/commit/dcb01ec2ae6843728031462161affe46c15d693e">dcb01ec2ae6843728031462161affe46c15d693e</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-12.liferay.com/job/test-portal-source-format/5742/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/5742/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-12.liferay.com/job/test-portal-source-format/5742/">test-portal-source-format#5742</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-12.liferay.com/userContent/jobs/test-portal-source-format/builds/5742/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-echo/liferay-portal/pull/14752">liferay-echo#14752</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2477154555">[master] ci:test:sf - veroglez &gt; liferay-echo - PR#14752 - 2023-12-20[03:39:01]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-12.liferay.com/job/publish-testray-report/4481/">publish-testray-report#4481</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 31 out of 31 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 67 out of 68 jobs passed in 1 hour 24 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/b808cb100001223008944254d5da1364de64c622">b808cb100001223008944254d5da1364de64c622</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/966339f8ff21668dccc44bb6674d0c0ead53d0e3">966339f8ff21668dccc44bb6674d0c0ead53d0e3</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2476996988">EE Development Acceptance (master) - 258 - 2023-12-19[16:42:54]</a></p><details><summary><strong>ci:test:stable - 31 out of 31 jobs PASSED</strong></summary><details><summary><strong>31 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 66 out of 68 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/8073/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/976221/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/976226/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-mysql57-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/757973/">empty-osgi-core-dir-mysql57-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-db2115-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/814481/">functional-smoke-tomcat90-db2115-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-hypersonic20-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/871393/">functional-smoke-tomcat90-hypersonic20-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/739746/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/976225/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-tomcat90-oracle193-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/814482/">functional-smoke-tomcat90-oracle193-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/974460/">functional-smoke-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql153-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/470452/">functional-smoke-tomcat90-postgresql153-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-sqlserver2019-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/786251/">functional-smoke-tomcat90-sqlserver2019-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/974463/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/470447/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-chrome100-jdk8<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312092/">functional-tomcat90-mysql57-chrome100-jdk8/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/470451/">functional-tomcat90-mysql57-chrome100-jdk8/0/1</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/704260/">functional-tomcat90-mysql57-chrome100-jdk8/0/2</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/863831/">functional-tomcat90-mysql57-chrome100-jdk8/0/3</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/757975/">functional-tomcat90-mysql57-chrome100-jdk8/0/4</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/849051/">functional-tomcat90-mysql57-chrome100-jdk8/0/5</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/863830/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/470450/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/496388/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/739748/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/786255/">functional-upgrade-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/704259/">functional-upgrade-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/893509/">functional-upgrade-tomcat90-mysql57-jdk8/0/6</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/470448/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/757972/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/496387/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/863829/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/802796/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/4</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/786254/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/5</a></li></ul></li><li>functional-upgrade-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/470449/">functional-upgrade-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-postgresql153-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/802797/">functional-upgrade-tomcat90-postgresql153-jdk8/0/0</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/849049/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-mysql57-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/871391/">lpkg-controller-mysql57-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/976222/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/976223/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/849046/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/976224/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/871392/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/786253/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/496386/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/863828/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/849047/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/849048/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/802795/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/786252/">modules-compile-jdk8/0/11</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/780199/"><strike>modules-integration-mysql57-jdk8/0/0</strike></a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/786257/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/871396/">modules-integration-mysql57-jdk8/0/2</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/802799/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/974465/">modules-integration-mysql57-jdk8_stable/0/1</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/849050/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/802798/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/893510/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/871390/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/974462/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-mysql57-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/849045/">patching-tool-mysql57-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/786256/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-mysql57-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/974464/">portal-license-smoke-mysql57-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/871395/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/739747/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/757971/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/757974/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/871394/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/974461/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></div><details><summary><strong>66 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8073/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 966339f8ff21668dccc44bb6674d0c0ead53d0e3:</strong></summary><ol><li><div><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/780199//consoleText">modules-integration-mysql57-jdk8/0/0</a><ol><li><details><summary><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/780199/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/780199//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li></ol></details></details></html>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/8073//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/145497
[Console](https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/8073//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/8073/">test-portal-acceptance-pullrequest(master)#8073</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8073/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-echo/liferay-portal/pull/14752">liferay-echo#14752</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2477162332">[master] ci:test:relevant - veroglez &gt; liferay-echo - PR#14752 - 2023-12-20[03:34:42]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/16005/">publish-testray-report#16005</a><br/></div></div>
